docs/migrating/migrating_from_ez_publish_platform.md

```diff
@@ -212,7 +220,7 @@ ImagesClassList[]=image
 
     Version of the migration script in ezplatform-xmltext-fieldtype prior to v1.6.0 would fail to convert embedded images correctly. If you have a database which you have already converted with an old version, you may rerun the `convert-to-richtext` command with the following options:
 
-    `php app/console ezxmltext:convert-to-richtext --fix-embedded-images-only --image-content-types=image,thumbnail,profile -v`
+    `php app/console ezxmltext:convert-to-richtext --fix-embedded-images-only --image-content-types=image -v`
```

Comment:
image is default right? So maybe split in two examples here, one if you only have image, and a second example just to show how you would specify image content types.

Reply:
Did some changes in f602589b86bd8120a03a5e068c3ae8b89c265bc3, ok?